portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,11 +32,27 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private final List<String> data = new ArrayList<>(); 
-
+  
+  private List<Comment> data;
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+     data = new ArrayList<>(); 
+    
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
```

Comment:
This means you are always getting all the comments -- not necessary fix this in this PR, but if your app has thousands of comments it might takes a long time to fetch the data and render the page.

Reply:
ohh yes makes sense. for step 6 I noticed that you let the user choose how many comments to display. I did it as a select menu, so I could only get the max number in the select menu? (could be a part of next PR)

Reply:
Sounds good to make this part of next PR